eppo_core/src/ufc/models.rs

```diff
@@ -18,6 +18,7 @@ pub type Timestamp = chrono::DateTime<chrono::Utc>;
 pub(crate) struct UniversalFlagConfigWire {
     /// When configuration was last updated.
     pub created_at: Timestamp,
+    pub format: AssignmentFormat,
```

Comment:
minor: ideally, this field should be optional because there could exist configurations without it (remember that users may be storing and passing configurations out of band now)

Reply:
https://github.com/Eppo-exp/eppo-multiplatform/pull/76